covenant-signer/docs/transition-from-phase1.md

```diff
@@ -0,0 +1,402 @@
+
+```shell
+    address: bbn1azasawj3ard0ffwj04zpxlw2pt9cp7kwjcdqmc
+    name: cov
+    pubkey: '{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"Ajp5tUbHnX98X/IGINkUtc9yUGMdEvbiZCftnT+Yxcyx"}'
+    type: local
+
+```
+
+Congratulations! You have successfully imported your key.
+
+## 4. Operation
+### 4.1. Configuration
+
+Next, we can return to the covenant signer directory 
+and create your own configuration file.
+
+Use the example configuration [file](../example/config.toml) to create your own 
+configuration file. Then, replace the placeholder values with your own 
+configuration. This can be placed directly in the `covenant-signer` directory.
+
+```toml
+[keystore]
+keystore-type = "cosmos"
+
+[keystore.cosmos]
+# pointing to the directory where the key is stored, unless specified otherwise
+key-directory = "/path/to/keydir"
+
+# the backend to be used for storing the key, in this case file
+keyring-backend = "file"
+
+# the name of the key you used when importing the key
+key-name = "your-key-name"
+
+# the chain id of the chain to be used
+chain-id = "your-chain-id"
+
+[server-config]
+# The IP address where the covenant-signer server will listen
+host = "127.0.0.1"
+# The TCP port number where the covenant-signer server will listen
+port = 9791
+
+[metrics]
+# The IP address where the Prometheus metrics server will listen
+host = "127.0.0.1"
+# This port is used to expose metrics that can be scraped by Prometheus
+port = 2113
+```
+
+Below are brief explanations of the configuration entries:
+
+- `keystore-type`: Type of keystore used, which is "cosmos"
+- `key-directory`: Path where keys are stored on the filesystem.
+- `keyring-backend`: Backend system for key management, e.g., "file", "os".
+- `key-name`: Name of the key used for signing transactions.
+- `chain-id`: Identifier of the blockchain network.
+- `host` (server-config): IP address where the server listens, typically "127.0.0.1" for local access.
+- `port` (server-config): TCP port number for the server.
+- `host` (metrics): IP address for the Prometheus metrics server, typically "127.0.0.1".
+- `port` (metrics): TCP port number for the Prometheus metrics server.
+
+### 4.2. Starting the daemon
+
+We then will run the following command to start the daemon from the 
+`covenant-signer` directory:
+
+```shell
+covenant-signer start --config ./path/to/config.toml
+```
+
+The covenant signer must be run in a secure network and only accessible by the 
+covenant emulator.
+
+Once the covenant signer is set up and unlocked, you can configure the covenant 
+emulator to use it. The URL of the covenant signer is configurable (`remotesigner` section)
+but in this example we use the default value of 
+`http://127.0.0.1:9791`.
+
+### 4.3. Unlocking the key
+
+Before you can sign transactions with the covenant key, you must unlock the 
+keyring that stores it. This happens through a `POST` request
+on the `v1/unlock` endpoint with a payload containing
```

Comment:
I guess we need a CLI command on performing this as well cc @KonradStaniec

Reply:
humh yup this is for operator of the program so 👍

Reply:
https://github.com/babylonlabs-io/covenant-emulator/issues/61